.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.4
```

Comment:
I added it because we have it in propsd as well as for when we add in the ruby authenticate pice.